class UserAlreadyExist(Exception):
    pass

class InvalidadData(Exception):
    pass